MetricsDashboard.Core/ITileRepository.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using MetricsDashboard.Core.Entities;
+using MetricsDashboard.Core.Models;
+
+namespace MetricsDashboard.Core
+{
+    public interface ITileRepository
+    {
+        Task<TileEntity> GetTileAsync(string tileName, CancellationToken cancellationToken);
```

Comment:
If we want to use Domain Models then repositories should return DomainModels, and MongoEntities should be used only inside of repositories.

Reply:
Thanks to that we will separate Domain Code from Data Access Code.

Reply:
Now repository returns only Entities